test/mlir/onnx/onnx_canonicalization.mlir

```diff
@@ -191,8 +191,6 @@ func @test_conv_batchnormtestmode_fusion(%arg0 : tensor<1x3x224x224xf32>, %arg1
     // CHECK: return [[RES]] : tensor<1x64x112x112xf32>
 }
 
```

Comment:
I thought the 
```
// -------
```
were required to get some parallelism. Please add back unless it was discontinued